homeassistant/components/bom/weather.py

```diff
@@ -54,7 +54,9 @@ def name(self):
     @property
     def condition(self):
         """Return the current condition."""
-        return self.bom_data.get_reading("weather")
+        return self.bom_data.get_reading("weather") or self.bom_data.get_reading(
+            "cloud"
+        )
```

Comment:
As mentioned in the linked issue, does it make sense to fall back to "Clear" if both these are None?

Reply:
I'm hesitant to do this, as the weather attribute lists `"weather": "Fine"` and I have seen a `Clear` state in the past. 

For certain regions, BOM provides no `weather` attribute updates for several days. In this case, isn't a better option to show `unknown` instead of `Clear`? Otherwise if BOM, hasn't reported a new `weather` state in a region, it will display `Clear` even when it may be raining?

I have attempted to contact BOM for clarification on this, but I haven't received a reply. I think automatically defaulting to `"-"` may result in the wrong forecast in some contexts. 

This scenario may be rare, but it depends on how important the forecast accuracy should be, I think its better to report `unknown` in this context, rather than infer it may be `Clear` ?